Sources/SourceKit/sourcekitd/CursorInfo.swift

```diff
@@ -33,10 +33,14 @@ struct CursorInfo {
   /// https://github.com/apple/swift/blob/master/bindings/xml/comment-xml-schema.rng
   var documentationXML: String?
 
-  init(_ symbolInfo: SymbolDetails, annotatedDeclaration: String?, documentationXML: String?) {
+  /// The refactor actions available at this position.
+  var refactorActions: SKResponseArray? = nil
```

Comment:
All the other data here we are pre-converting - e.g. the SymbolDetails.  What if we converted to an array of `[SemanticRefactorCommand]` before putting it in CursorInfo?